contracts/Core/BlockManager.sol

```diff
@@ -99,12 +99,12 @@ contract BlockManager is ACL, BlockStorage {
         ) 
     {
         _block = proposedBlocks[epoch][proposedBlock];
-        return(_block, _block.medians, _block.lowerCutoffs, _block.higherCutoffs);
+        return(_block, _block.aggregate, _block.lowerCutoffs, _block.higherCutoffs);
     }
 
     function getProposedBlockMedians(uint256 epoch, uint256 proposedBlock)
```

Comment:
Shouldn't we rename this function as well to aggregates?

Reply:
Yes. Will make this change